js-callback-functions_user-login-callback-2-with-argument/js/index.js

```diff
@@ -0,0 +1,17 @@
+console.clear();
+
+function handleUserLogin(onSuccess) {
+  onSuccess("Jane Doe");
+}
+
+// The exercise starts here!
+function showWelcomeMessage(userName) {
+  console.log("Welcome Jane Doe! You are logged in now.")
```

Comment:
I think we are supposed to pass the userName as an argument ie (`Welcome ${userName}! You are logged in now.`). It logs the same.

Reply:
Danke!